posthog/api/test/dashboards/__snapshots__/test_dashboard.ambr

```diff
@@ -2777,7 +2777,7 @@
                                                           2,
                                                           3,
                                                           4,
-                                                          5 /* ... */) /*controller='project_dashboards-list',route='api/projects/%28%3FP%3Cparent_lookup_team_id%3E%5B%5E/.%5D%2B%29/dashboards/%3F%24'*/
+                                                          5 /* ... */)
```

Comment:
is this just flakiness? I think your other PR had something similar

Reply:
ugh... seems so

Reply:
django 4.2 upgrade removed the sql commenter sadly, and it seems like this snapshot didn't update then, which is why.

If you see a test that brings these comments back, let me (or Julian really) know, that shouldn't happen!